benchmark/README.md

```diff
@@ -2,7 +2,28 @@ Benchmarking
 ===
 
 # Generate Data
-> *TODO* instructions to be provided
+1. [Download](http://www.tpc.org/tpc_documents_current_versions/current_specifications.asp) the TPC-H benchmark tool.
+Follow the instructions for registration and download the tool and copy to local disk with free space larger than 300GB.
+
+2. Build the dbgen tool.
+    - Decompress the zip file, then navigate to `dbgen` folder.
+    - The TPC-H README contains instructions on how to build the tool in Linux.
+    - For Windows users, you can generate the dbgen.exe using Visual Studio: 
+        - i. In the `dbgen` folder, you will see `tpch.sln`, open it using Visual Studio (VS2015 or VS2017 should work).
+        - ii. Click on tab “Build”, then “Build Solution”. 
+        - iii. Once the build is successful, it should generate `dbgen.exe` and `qgen.exe` in the `Debug` folder.
+
+3. Generate the data.
+    - The TPC-H README contains instructions on how to generate a database population in Linux OS.
+    - For Windows users:
+        - i. Copy `dbgen.exe` to the `dbgen` folder
+        - ii. Open cmd terminal, navigate to `dbgen` folder, and type command: `dbgen -vf -s 300`, which is generating 300GB TPCH dataset. It could take up to 40 hours to complete. Note that you can use command: `dbgen -h`, to check all the options. 
+    - After database population generation is completed, you will see 8 tables (customer, lineitem, nation, orders, part, partsupp, region, supplier) in tbl format:
+
+
+4. Convert TPCH dataset to parquet format. 
+You can write your own code or copy paste a simple Spark application [code](https://github.com/laserljy/spark-1/commit/c01ba709f7797e55ed96ff4fcd23f090f26b5bcf), and run the job to convert the TPCH dataset to parquet format. 

```

Comment:
You can write your own application or use [this Spark application](...) as follow:
```
how to run your app here
```

Reply:
I ran the in IntelliJ, but have not tried running the job via spark-sumbit. So how about adding that "Copy paste the provided code in IntelliJ to run the job"?

Reply:
Check #62 for usage I put. Since this is part of benchmark project, you can just build the package and pass the jar to the spark-submit (and get rid of "you can write your own code")